_episodes_rmd/03-data-structures-part1.Rmd

```diff
@@ -48,6 +48,12 @@ them using the `$` operator:
 nordic$country
 nordic$lifeExp
 ```
+Note that when you type '$' after the data frame variable name, the column names will pop up as suggestions. You can also explore the data frame using 'r head' and 'r names' functions:
+
+'''{r}
+head(nordisc)
+names(nordisc)
+'''
```

Comment:
I suppose this should be within backticks as ` ```{r} ` and ` ``` ` instead of using quotes ` '''{r} ` and ` ''' `